src/build-task/AugurkCSharpAnalyzer/task.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "3b04573e-7699-440a-93ae-8f7b06259078",
+  "name": "AugurkCSharpAnalyzer",
+  "friendlyName": "Run Augurk C# Analysis",
+  "description": "Performs a static code analysis over your test automation and uploads the results to Augurk",
```

Comment:
Shouldn't we add the reason why you would want this to the description? In a single line of course.